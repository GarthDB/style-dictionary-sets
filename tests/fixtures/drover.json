{
  "black": {
    "value": "rgb(0, 0, 0)"
  },
  "blackAlias": {
    "value": "{black}"
  },
  "neutral-background-color-default": {
    "sets": {
      "light": {
        "value": "{gray-800}"
      },
      "dark": {
        "value": "{gray-400}"
      },
      "darkest": {
        "value": "{gray-400}"
      },
      "wireframe": {
        "value": "{gray-800}"
      }
    }
  },
  "accent-color-100": {
    "sets": {
      "spectrum": {
        "value": "{gray-800}"
      },
      "express": {
        "value": "{gray-400}"
      }
    }
  },
  "gray-400": {
    "sets": {
      "light": {
        "value": "rgb(177, 177, 177)"
      },
      "dark": {
        "value": "rgb(112, 112, 112)"
      },
      "darkest": {
        "value": "rgb(106, 106, 106)"
      },
      "wireframe": {
        "value": "rgb(183, 200, 235)"
      }
    }
  },
  "gray-800": {
    "sets": {
      "light": {
        "value": "rgb(34, 34, 34)"
      },
      "dark": {
        "value": "rgb(235, 235, 235)"
      },
      "darkest": {
        "value": "rgb(235, 235, 235)"
      },
      "wireframe": {
        "value": "rgb(73, 110, 194)"
      }
    }
  },
  "component-pill-edge-to-visual-100": {
    "sets": {
      "desktop": {
        "value": "17px"
      },
      "mobile": {
        "value": "17px"
      }
    }
  },
  "corner-radius-75": {
    "sets": {
      "spectrum": {
        "sets": {
          "desktop": {
            "value": "2px"
          },
          "mobile": {
            "value": "2px"
          }
        }
      },
      "express": {
        "sets": {
          "desktop": {
            "value": "3px"
          },
          "mobile": {
            "value": "4px"
          }
        }
      }
    }
  },
  "base-corner-radius": {
    "value": "2px"
  },
  "another-corner-radius": {
    "value": "{base-corner-radius}"
  },
  "ref-express-set":{
    "value": "{corner-radius-75}"
  },
  "android-elevation": {
    "value": "2dp"
  },
  "transparent-black-200": {
    "value": "rgba(0, 0, 0, 0.10)"
  }
}
