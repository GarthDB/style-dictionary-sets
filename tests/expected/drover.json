{
  "drover": {
    "system": "Spectrum",
    "version": "11.7.0",
    "KEYS": {
      "DIMENSION_LAYOUT_TOKENS": "layoutTokens",
      "DIMENSION_COMPONENT_LAYOUT_TOKENS": "componentLayoutTokens",
      "COLOR_THEMES": "colorThemes",
      "COLOR_TOKENS": "colorTokens",
      "COLOR_ALIASES": "colorAliases",
      "COLOR_SEMANTICS": "colorSemantics",
      "DIMENSIONS": "dimensions"
    },
    "colorThemes": {
      "light": {
        "accent-color-100": "rgb(177, 177, 177)",
        "black": "rgb(0, 0, 0)",
        "blackAlias": "rgb(0, 0, 0)",
        "gray-400": "rgb(183, 200, 235)",
        "gray-800": "rgb(73, 110, 194)",
        "neutral-background-color-default": "rgb(34, 34, 34)"
      },
      "dark": {
        "accent-color-100": "rgb(112, 112, 112)",
        "black": "rgb(0, 0, 0)",
        "blackAlias": "rgb(0, 0, 0)",
        "gray-400": "rgb(183, 200, 235)",
        "gray-800": "rgb(73, 110, 194)",
        "neutral-background-color-default": "rgb(235, 235, 235)"
      },
      "darkest": {
        "accent-color-100": "rgb(106, 106, 106)",
        "black": "rgb(0, 0, 0)",
        "blackAlias": "rgb(0, 0, 0)",
        "gray-400": "rgb(183, 200, 235)",
        "gray-800": "rgb(73, 110, 194)",
        "neutral-background-color-default": "rgb(235, 235, 235)"
      }
    },
    "dimensions": {
      "description": "A scale stop of a given size",
      "scale-factor": "1",
      "base-corner-radius": "2px",
      "component-pill-edge-to-visual-100": "17px",
      "corner-radius-75": "2px"
    }
  }
}
